### ISSUE TITLE
Chopping block error

### ISSUE THREAD

**Issue by darkhax** [Darkosto/SevTech-Ages#98]
### Issue / bug
This error is put into the server console while a chopping block is being used. 
```
java.util.concurrent.ExecutionException: java.lang.NoClassDefFoundError: net/minecraft/client/Minecraft
at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_121]
at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_121]
at net.minecraft.util.Util.func_181617_a(SourceFile:47) [h.class:?]
at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:721) [MinecraftServer.class:?]
at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:396) [nz.class:?]
at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:666) [MinecraftServer.class:?]
at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:524) [MinecraftServer.class:?]
at java.lang.Thread.run(Thread.java:745) [?:1.8.0_121]
Caused by: java.lang.NoClassDefFoundError: net/minecraft/client/Minecraft
at se.gory_moon.horsepower.util.Utils.getTextureFromBlockstate(Utils.java:105) ~[Utils.class:?]
at se.gory_moon.horsepower.blocks.BlockHPChoppingBase.getExtendedState(BlockHPChoppingBase.java:178) ~[BlockHPChoppingBase.class:?]
at se.gory_moon.horsepower.blocks.BlockHPChoppingBase.getExtendedState(BlockHPChoppingBase.java:51) ~[BlockHPChoppingBase.class:?]
at se.gory_moon.horsepower.blocks.BlockHPBase.func_180639_a(BlockHPBase.java:183) ~[BlockHPBase.class:?]
at se.gory_moon.horsepower.blocks.BlockChoppingBlock.func_180639_a(BlockChoppingBlock.java:59) ~[BlockChoppingBlock.class:?]
at net.minecraft.server.management.PlayerInteractionManager.func_187251_a(PlayerInteractionManager.java:446) ~[or.class:?]
at net.minecraft.network.NetHandlerPlayServer.func_184337_a(NetHandlerPlayServer.java:731) ~[pa.class:?]
at net.minecraft.network.play.client.CPacketPlayerTryUseItemOnBlock.func_148833_a(SourceFile:55) ~[ma.class:?]
at net.minecraft.network.play.client.CPacketPlayerTryUseItemOnBlock.func_148833_a(SourceFile:11) ~[ma.class:?]
at net.minecraft.network.PacketThreadUtil$1.run(SourceFile:13) ~[hv$1.class:?]
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_121]
at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_121]
at net.minecraft.util.Util.func_181617_a(SourceFile:46) ~[h.class:?]
```

**Comment by Darkosto** [comment]
I'll tag @gorymoon for this one to see what he thinks.

**Comment by GoryMoon** [comment]
Can't reproduce it, but have a fix that should prevent it from happening, fixed in 2.5.7.51
